lua/wire/client/e2_extension_menu.lua

```diff
@@ -32,14 +30,14 @@ local function BuildExtensionMenu(panel)
     local checkbox = vgui.Create("DCheckBoxLabel", item)
     checkbox:SetText(name)
     checkbox:SetChecked(E2Lib.GetExtensionStatus(name))
-    checkbox.Button:SetDisabled(checkboxes_disabled)
+    checkbox.Button:SetDisabled(not bAllowed)
     checkbox:SizeToContents()
     checkbox:SetDark(true)
     function checkbox:OnChange(value)
       LocalPlayer():ConCommand(CONCOMMAND_NAMES[value] .. name)
     end
 
-    if not checkboxes_disabled then
+    if (not bAllowed) then
```

Comment:
This is broken - now this function is only set if the user's *not* allowed to click it.